app/src/main/java/org/stepic/droid/util/StringExtensions.kt

```diff
@@ -0,0 +1,16 @@
+package org.stepic.droid.util
+
+
+/*
+ * Counts while predicate is true
+ */
+inline fun String.countWhile(start: Int = 0, predicate: (Char) -> Boolean) : Int {
```

Comment:
lets rename `start` to `startIndex`, because countWhile.start may be also number of start counting